src/path.rs

```diff
@@ -204,7 +204,7 @@ impl ClioPath {
         self.with_path_mut(|path| path.set_extension(extension))
     }
 
-    /// Adds an extention to the end of the [`self.file_name`](Path::file_name).
+    /// Adds an extension to the end of the [`self.file_name`](Path::file_name).
```

Comment:
For once it looks like I spelled something correctly, it really is `extension` e.g. [`PathBuf::set_extension`](https://doc.rust-lang.org/std/path/struct.PathBuf.html#method.set_extension).

Reply:
you are right I should go get my eyes tested to see if I am colour blind.